tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/TestHelper.cs

```diff
@@ -373,7 +373,7 @@ public ProcessResult WaitForProcessResult(ProcessHelper helper)
 
         public void EnableIast(bool enable = true)
         {
-            SetEnvironmentVariable(ConfigurationKeys.Iast.Enabled, enable.ToString());
+            SetEnvironmentVariable(ConfigurationKeys.Iast.Enabled, enable.ToString().ToLower());
```

Comment:
This _shouldn't_ be required 🤔 We should accomodate additional cases. If this is about getting the variable in native code, there are helpers there to do this for you IIRC

Reply:
It seems that the helper that the native profiler is using (ToBooleanWithDefault) does not include "True". I could change it, but it would affect to many other parts of the native code... I will keep it as a task to discuss with the native developers.

Reply:
Cool, It would definitely be preferable to update `ToBooleanWithDefault` so that it matches our native code, but I agree, that's out of scope for this PR 🙂